src/engraving/accessibility/accessibleitem.cpp

```diff
@@ -49,8 +51,6 @@ AccessibleItem::~AccessibleItem()
         accessibilityController()->unreg(this);
```

Comment:
there is *still* undefined behaviour here, because unreg calls a virtual function on the passed-in argument
@Eism 
But...it never gets executed because accessibleRoot( ) will always return nullptr after setting m_element to nullptr, hence this code is pointless...